pkg/terraform/exec/plugins/vendor/github.com/terraform-providers/terraform-provider-azurerm/azurerm/resource_arm_virtual_network.go

```diff
@@ -416,8 +448,9 @@ func resourceAzureSubnetHash(v interface{}) int {
 
 	if m, ok := v.(map[string]interface{}); ok {
 		buf.WriteString(m["name"].(string))
-		buf.WriteString(m["address_prefix"].(string))
-
+		if v, ok := m["address_prefix"]; ok {
+			buf.WriteString(v.(string))
+		}
```

Comment:
Not sure what this hash gets used for but you might need to use the `address_prefixes` value here, to ensure it stays unique

Reply:
Apparently it is used to uniquely identify the resource. I've looked around at many of the other hashes and they leave out elements. However, I have updated it to use address_prefixes. I'm testing it and if all goes well I'll update the PR.